FrontEnd/scripts/works.js

```diff
@@ -11,6 +11,7 @@
 // fetchWorkData2((response) => {
 //   console.log(response)
 // })
+let works = null;
```

Comment:
You should comment to explain what do you use a global variable. 

Also, it's probably time to move out the code you have above.